ui/components/ui/box/box.stories.js

```diff
@@ -346,6 +346,12 @@ export const Color = (args) => {
       <Box {...args} padding={3} color={COLORS.INHERIT} {...args}>
         COLORS.INHERIT
       </Box>
+      <Box {...args} padding={3} color={COLORS.SEPOLIA_INVERSE} {...args}>
+        COLORS.SEPOLIA_INVERSE
+      </Box>
+      <Box {...args} padding={3} color={COLORS.GOERLI_INVERSE} {...args}>
+        COLORS.GOERLI_INVERSE
+      </Box>
```

Comment:
Can you add the network defaults as background colors as well?

```suggestion
      <Box {...args} padding={3} backgroundColor={COLORS.SEPOLIA} color={COLORS.SEPOLIA_INVERSE} {...args}>
        COLORS.SEPOLIA_INVERSE
      </Box>
      <Box {...args} padding={3} backgroundColor={COLORS.GOERLI} color={COLORS.GOERLI_INVERSE} {...args}>
        COLORS.GOERLI_INVERSE
      </Box>
```